bindings/java/src/org/sleuthkit/datamodel/WindowsAccountUtils.java

```diff
@@ -158,7 +253,7 @@ static boolean isWindowsSpecialSid(String sid) {
 			}
```

Comment:
The above logic where we look if the sid starts with a prefix seems redundant since the following step where we check if its s-1-5-80 - s-5-1-111 will catch these as well.